iot-hub/Samples/device/DeviceReconnectionSample/DeviceReconnectionSample.cs

```diff
@@ -18,11 +18,12 @@ public class DeviceReconnectionSample
         private const int TemperatureThreshold = 30;
 
         private static readonly TimeSpan s_sleepDuration = TimeSpan.FromSeconds(5);
-        private static readonly TimeSpan s_operationTimeout = TimeSpan.FromHours(1);
 
         private readonly object _initLock = new object();
+        private readonly object _connectionStatusLock = new object();
         private readonly List<string> _deviceConnectionStrings;
         private readonly TransportType _transportType;
+        private readonly ClientOptions _clientOptions = new ClientOptions { SdkAssignsMessageId = Shared.SdkAssignsMessageId.WhenUnset };
```

Comment:
SdkAssignsMessageId ? What does this indicate? I'd rename for readability

Reply:
It means that if you do not set a message Id on a message, the the sdk will assign one for you